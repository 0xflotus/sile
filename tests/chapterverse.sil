\begin[papersize=a7,class=bible]{document}
\script[src=packages/lorem]
\begin{script}
SILE.registerCommand("format-reference", function (o,c)
  local ref
  if c.b then
	ref =  c.b .. " " .. c.c .. ", " .. c.v
  else
  	ref =  c.c .. ", " .. c.v
  end
  SILE.typesetter:typeset(ref)
end)
SILE.registerCommand("bible:verse-number", function(o,c) end)
\end{script}
\save-book-title{Gn}
\save-chapter-number{1}
\verse-number{1}
\verse-number{2}
foo
\end{document}
