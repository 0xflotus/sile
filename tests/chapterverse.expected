<tests/chapterverse.sil>
Set paper size 	209.7637818	297.6377985
Begin page
Mx 	20.0695
My 	42.0943
Set font 	Gentium;10;400;;normal;;LTR
T	70 79 79	(foo)
Mx 	152.0892
My 	17.0138
T	39 78	(Gn)
Mx 	166.0795
T	17	(1)
Mx 	170.7719
T	12	(,)
Mx 	175.7045
T	18	(2)
[1] Mx 	96.5574
My 	262.9405
T	17	(1)
End page
Finish

